Polygon-Abstract/src/polygon/Triangle.java

```diff
@@ -0,0 +1,74 @@
+package polygon;
+
+import java.util.ArrayList;
+//import java.awt.Point;
+
+public class Triangle extends Polygon{
+
+	
+	public static final int NUMOFSIDES = 3;
+	
+	
+	
+	
+	public Triangle(ArrayList<Point> coors) {
+		super(NUMOFSIDES, coors);
+		
+	}
+	
+	public Triangle(Point p1, Point p2, Point p3) {
+		super(NUMOFSIDES);
+		addOneCoordinate(p1);
+		addOneCoordinate(p2);
+		addOneCoordinate(p3);
+	}
+
+	
+
+	@Override
+	public double area() {
+		// TODO Auto-generated method stub
+		
+		Point a = coordinates.get(0);
+		
+		Point b = coordinates.get(1);
+		
+		Point c = coordinates.get(2);
+		
+		double crossP = (b.x - a.x) * (c.y - a.y) - (b.y - a.y) * (c.x - a.x);
+		
+		return Math.abs(crossP / 2);
```

Comment:
Can try not to have space between lines every single time. We don't need to double space stuff in cs lol. Space is used to separate big chunks of code.